tests/lax_numpy_indexing_test.py

```diff
@@ -982,6 +982,42 @@ def testSegmentSum(self):
     expected = onp.array([13, 2, 7, 4])
     self.assertAllClose(ans, expected, check_dtypes=False)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_val={}_idx={}".format(
+          jtu.format_shape_dtype_string(data_shape, val_dtype),
+          jtu.format_shape_dtype_string(data_shape, idx_dtype)),
+       "val_dtype": val_dtype, "idx_dtype": idx_dtype,
+       "rng_idx_factory": rng_idx_factory,
```

Comment:
There's actually no need to make these test parameters if there's only one of them. (The reason many test cases put these as parameters is historical.)